src/Nethermind/Nethermind.Consensus/Processing/BlockCachePreWarmer.cs

```diff
@@ -75,8 +77,8 @@ void WarmupWithdrawals(ParallelOptions parallelOptions, IReleaseSpec spec, Block
                 Parallel.For(0, block.Withdrawals.Length, parallelOptions,
                     i =>
                     {
-                        IReadOnlyTxProcessorSource env = _envPool.Get();
-                        using IReadOnlyTxProcessingScope scope = env.Build(stateRoot);
+                        IReadOnlyTxProcessingScope scope = _envPool.Get();
```

Comment:
not disposing the scope?

Reply:
No, prewarmer is handled specially, the scope are long lived here.